dll/cpl/sysdm/lang/bg-BG.rc

```diff
@@ -325,6 +325,10 @@ END
 
 STRINGTABLE
 BEGIN
+    IDS_EDITUSERVARCAPTION "Edit user variable"
```

Comment:
Please move those new four strings at the end of the stringtable (where the other ones related to environment variables are already present).